include/PredictionManager.hpp

```diff
@@ -86,10 +87,9 @@ class PredictionBuilder {
             final_data.push_back(pair.second ? 1.0 : 0.0);
         }
         // now we normalize
```

Comment:
fake news, it is not normalized, this is false advertisement and you will therefore get sued if this isn't removed